src/Protocol/ActivityPub/Receiver.php

```diff
@@ -641,7 +642,7 @@ public static function processActivity(array $activity, string $body = '', int $
 
 		// Test the provided signatures against the actor and "attributedTo"
 		if ($trust_source) {
-			if (!empty($attributed_to) && !empty($actor)) {
+			if (!empty($attributed_to) && $actor !== null) {
```

Comment:
Here, `$actor` is always set and non-null because of the check on line 622. Not sure why PHPStan made you keep the condition.

Reply:
done.